modules/system/assets/ui/js/datepicker.js

```diff
@@ -316,7 +316,8 @@
         format: null,
         yearRange: 10,
         firstDay: 0,
-        showWeekNumber: false
+        showWeekNumber: false,
+        mode: 'datetime',
```

Comment:
@fansaien could you remove trailing comma.

Reply:
Sure, I submit that.